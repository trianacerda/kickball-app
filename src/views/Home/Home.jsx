import { Link } from 'react-router-dom';
import React from 'react';

function Home() {
  return (
    <>
      <Link to="/teams" className="teams-link" exact>
        Teams
      </Link>
      <Link to="/players" className="player-link" exact>
        Players
      </Link>
      <h1>Hi from the home page!!!!!</h1>
    </>
  );
}

export default Home;
