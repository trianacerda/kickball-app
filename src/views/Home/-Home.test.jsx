import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Home from './Home';

it('should render the home page', async () => {
  const { container } = render(
    <MemoryRouter>
      <Home />
    </MemoryRouter>
  );

  const homePage = await screen.findByText('Hi from the home page!!!!!');
  const kickball = await screen.findByText('Kickball', { exact: false });

  expect(homePage, kickball).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
