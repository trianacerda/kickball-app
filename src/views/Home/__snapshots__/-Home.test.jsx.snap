// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the home page 1`] = `
<div>
  <a
    class="teams-link"
    href="/teams"
    style="display: flex; flex-direction: start;"
  >
    Teams
  </a>
  <a
    class="player-link"
    href="/players"
    style="display: flex; flex-direction: start;"
  >
    Players
  </a>
  <h1>
    Hi from the home page!!!!!
  </h1>
</div>
`;
